runtime/controller_test.go

```diff
@@ -569,7 +570,8 @@ path: data/foo.csv
 	testruntime.RequireReconcileState(t, rt, id, 3, 0, 0)
 	modelRes.Meta.Name.Name = "bar_new"
 	modelRes.Meta.FilePaths[0] = "/models/bar_new.sql"
-	model.State.Table = "bar_new"
+	model.State.ResultProperties = must(structpb.NewStruct(map[string]any{"table": "bar_new", "used_model_name": true, "view": true}))
```

Comment:
**Issue**: The use of 'must' for error handling in test setups is risky. It can cause the test suite to panic without giving a clear indication of what went wrong during the setup phase. <br> **Fix**: Replace 'must' with proper error handling. For example, use a helper function that handles errors gracefully by failing the test instead of causing a panic. <br> **Code Suggestion**: 
 ```
 - model.State.ResultProperties = must(structpb.NewStruct(map[string]any{"table": "bar_new", "used_model_name": true, "view": true}))
 + model.State.ResultProperties, err = structpb.NewStruct(map[string]any{"table": "bar_new", "used_model_name": true, "view": true})
 + if err != nil {
 +     t.Fatalf("Failed to set ResultProperties: %v", err)
 + }
 ```